react/components/NetworkStatusToast.js

```diff
@@ -52,10 +52,7 @@ function NetworkStatusToast(props) {
 }
 
 NetworkStatusToast.propTypes = {
-  hideToast: PropTypes.func.isRequired,
   intl: intlShape.isRequired,
```

Comment:
shouldnt you just remove the `isRequired` flag? are these props just not used anymore?

Reply:
They don't come from props anymore, they come from the hook ``useContext`